R/rules.R

```diff
@@ -295,3 +294,93 @@ sv_between <- function(left,
     }
   }
 }
+
+#' Validate that a field is part of a defined set
+#'
+#' The `sv_in_set()` function checks whether the field value is part of a
+#' specified set of values.
+#' 
+#' @param set A vector of elements for which the field value must be a part of
```

Comment:
"A vector or list of elements"
Also note that NA and NaN can be included if desired

Reply:
Changed now.